### PR TITLE
Added reference to KB article

### PR DESCRIPTION
**Pull request by Batre-MSFT**
Added reference to KB article for multi AD domain environment
https://docs.microsoft.com/en-us/exchange/troubleshoot/public-folders/migrationbatch-fails-no-public-folder-mailboxes

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer/collaboration/public-folders/migrate-to-exchange-online.md [hunk 1]
```diff
@@ -456,7 +456,10 @@ You need to check the following items before you can complete your public folder
    .\Sync-ModernMailPublicFolders.ps1 -Credential (Get-Credential) -CsvSummaryFile:sync_summary.csv
    ```
 
-3. To complete the public folder migration, run the following command in Exchange Online PowerShell:
+3. If your environment has multiple active directory domains, ensure the steps in the following article are followed before initiating completing.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/collaboration/public-folders/migrate-to-exchange-online.md:459]:
```suggestion
3. If your environment has multiple active directory domains, ensure the steps in the following article are followed before initiating completion:
```

File: Exchange/ExchangeServer/collaboration/public-folders/migrate-to-exchange-online.md [hunk 2]
```diff
@@ -456,7 +456,10 @@ You need to check the following items before you can complete your public folder
...
+https://docs.microsoft.com/en-us/exchange/troubleshoot/public-folders/migrationbatch-fails-no-public-folder-mailboxes
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/collaboration/public-folders/migrate-to-exchange-online.md:460]:
```suggestion
["No active public folder mailboxes were found" error and migration batch fails at Complete-MigrationBatch command](https://docs.microsoft.com/exchange/troubleshoot/public-folders/migrationbatch-fails-no-public-folder-mailboxes)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**